app/src/main/java/org/thoughtcrime/securesms/conversation/v2/ConversationViewModel.kt

```diff
@@ -778,6 +910,37 @@ class ConversationViewModel(
         }
     }
 
+    fun onOptionItemSelected(
+        // This must be the context of the activity as requirement from ConversationMenuHelper
+        context: Context,
```

Comment:
Isn't there a strict rule around passing activity context to a VM? This feels like a memory leak scenario

Reply:
So `ConversationMenuHelper` needs activity context and it doesn't really keep the context around, from reading the code. TBH `ConversationMenuHelper` is a giant mess and I don't think I want to refactor it now.